bin/ci2gubernator.sh

```diff
@@ -68,5 +68,9 @@ if [ -n "$CIRCLE_PR_NUMBER" ] || [ -n "$CIRCLE_PULL_REQUEST" ]; then
 	ARGS+=("--stage=presubmit")
 fi
 
+if [ ${CIRCLE_BRANCH} == "master" ]; then
```

Comment:
At some point, will we have this working for all branches?

Reply:
The only problem with having it for all branches testgrid config. It will be a lot of copy pasting, which might create unexpected errors. But I will set it up for release 1.2 and future releases. I expect  devs who is working on feature branches to update testgrid  themselves.

Reply:
I think it is fine to manually update testgrid for feature branches.